src/tests/functional/plugin/conformance/test_runner/op_conformance_runner/src/read_ir/read_ir.cpp

```diff
@@ -11,6 +11,7 @@
 #include "common_test_utils/postgres_link.hpp"
 
 #include "shared_test_classes/base/utils/generate_inputs.hpp"
+#include "shared_test_classes/base/utils/generate_static_shapes.hpp"
```

Comment:
Let's move the file to `src/tests/functional/plugin/conformance/test_runner/op_conformance_runner/include/utils` because we will use it only for conformance tests

Reply:
Moved to `src/tests/functional/plugin/conformance/test_runner/op_conformance_runner/include/utils`. Thank you